libs/langchain/langchain/callbacks/tracers/base.py

```diff
@@ -49,7 +49,7 @@ def _persist_run(self, run: Run) -> None:
     def _start_trace(self, run: Run) -> None:
         """Start a trace for a run."""
         if run.parent_run_id:
-            parent_run = self.run_map[str(run.parent_run_id)]
+            parent_run = self.run_map.get(str(run.parent_run_id))
```

Comment:
Did this break for an example?

Reply:
Yes when I accidentally had an error in my example, in addition to the error was seeing this key error printed